lib/Experian/IDAuth.pm

```diff
@@ -152,19 +155,42 @@ sub _build_request {
     my $self = shift;
 
     $self->{request_as_xml} =
-          '<Search>'
+          '<xml><![CDATA[<Search>'
         . $self->_build_authentication_tag
         . $self->_build_country_code_tag
         . $self->_build_person_tag
         . $self->_build_addresses_tag
         . $self->_build_telephones_tag
         . $self->_build_search_reference_tag
         . $self->_build_search_option_tag
-        . '</Search>';
+        . '</Search>]]></xml>';
```

Comment:
Why are we putting thing inside `CDATA` now?

Reply:
SOAP::Lite handled formatting by itself when we used to pass in a single string containing the request. But since we now need to pass in a header as well, we need to handle formatting both the body and header ourselves.